VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimsAndAppealsListView/ClaimsAndAppealsListView.test.tsx

```diff
@@ -123,83 +115,42 @@ context('ClaimsAndAppealsListView', () => {
         },
       },
     })
-
-    testInstance = component.UNSAFE_root
   }
 
   beforeEach(async () => {
     initializeTestInstance('ACTIVE')
   })
 
   it('initializes correctly', async () => {
-    expect(component).toBeTruthy()
-  })
-
-  describe('when the claimType is ACTIVE', () => {
-    it('should display the header as "Your active claims and appeals"', async () => {
-      expect(testInstance.findAllByType(TextView)[0].props.children).toEqual('Your active claims and appeals')
-    })
-  })
-
-  describe('when the claimType is CLOSED', () => {
-    it('should display the header as "Your closed claims and appeals"', async () => {
-      await initializeTestInstance('CLOSED')
-      expect(testInstance.findAllByType(TextView)[0].props.children).toEqual('Your closed claims and appeals')
-    })
-  })
-
-  describe('when an item is type claim', () => {
-    it('should display the first line with the format "Claim for {{subtype}} updated on MMMM, dd yyyy"', async () => {
-      expect(testInstance.findAllByType(TextView)[3].props.children).toEqual('Claim for compensation updated on October 30, 2020')
-    })
-
-    it('should display the second line as "Submitted on MMMM dd, yyyy', async () => {
-      expect(testInstance.findAllByType(TextView)[4].props.children).toEqual('Submitted October 22, 2020')
-    })
-  })
-
-  describe('when an item is type appeal', () => {
-    it('should display the first line with the format "{{subtype}} updated on MMMM, dd yyyy"', async () => {
-      expect(testInstance.findAllByType(TextView)[1].props.children).toEqual('Supplemental claim for disability compensation updated on October 28, 2020')
-    })
-
-    it('should display the second line as "Submitted on MMMM dd, yyyy', async () => {
-      expect(testInstance.findAllByType(TextView)[2].props.children).toEqual('Submitted October 22, 2020')
-    })
+    expect(screen.getByText('Your active claims and appeals')).toBeTruthy()
+    expect(screen.queryByText('Your closed claims and appeals')).toBeFalsy()
+    expect(screen.getByText('Supplemental claim for disability compensation updated on October 28, 2020')).toBeTruthy()
+    expect(screen.getAllByText('Submitted October 22, 2020')).toBeTruthy()
+    expect(screen.getByText('Claim for compensation updated on October 30, 2020')).toBeTruthy()
+    initializeTestInstance('CLOSED')
+    expect(screen.getByText('Your closed claims and appeals')).toBeTruthy()
+    expect(screen.queryByText('Your active claims and appeals')).toBeFalsy()
   })
 
   describe('on click of a claim', () => {
     it('should call useRouteNavigation', async () => {
-      testInstance.findAllByType(Pressable)[1].props.onPress()
+      fireEvent.press(screen.getByTestId('Claim for compensation updated on October 30, 2020 Submitted October 22, 2020'))
```

Comment:
Using the preferred `getByRole`
```suggestion
      fireEvent.press(screen.getByRole('button', { name: 'Claim for compensation updated on October 30, 2020 Submitted October 22, 2020' }))
```

Reply:
updated